tests/crypto/test_cases/test_aead.c

```diff
@@ -0,0 +1,575 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <stdio.h>
+#include <stdarg.h>
+#include <string.h>
+#include <stddef.h>
+#include <logging/log.h>
+
+#include "common_test.h"
+#include <mbedtls/ccm.h>
+
+LOG_MODULE_REGISTER(test_aead, LOG_LEVEL_INF);
+
+#if defined(MBEDTLS_AES_C)
+
+extern test_vector_aead_t __start_test_vector_aead_ccm_data[];
+extern test_vector_aead_t __stop_test_vector_aead_ccm_data[];
+extern test_vector_aead_t __start_test_vector_aead_ccm_simple_data[];
+extern test_vector_aead_t __stop_test_vector_aead_ccm_simple_data[];
+
+extern test_vector_aead_t __start_test_vector_aead_gcm_data[];
+extern test_vector_aead_t __stop_test_vector_aead_gcm_data[];
+extern test_vector_aead_t __start_test_vector_aead_gcm_simple_data[];
+extern test_vector_aead_t __stop_test_vector_aead_gcm_simple_data[];
+
+extern test_vector_aead_t __start_test_vector_aead_chachapoly_data[];
+extern test_vector_aead_t __stop_test_vector_aead_chachapoly_data[];
+extern test_vector_aead_t __start_test_vector_aead_chachapoly_simple_data[];
+extern test_vector_aead_t __stop_test_vector_aead_chachapoly_simple_data[];
+
+#define NUM_BUFFER_OVERFLOW_TEST_BYTES (2)                                                      /**< Number of bytes to be used in the overflow test for AEAD. */
+#define AEAD_MAC_SIZE                  (16)                                                     /**< Size of the AEAD MAC. */
```

Comment:
Comments that are just repeating the variable name should be omitted.

Reply:
Several such instances removed in 7aee7a9